.circleci/config.yml

```diff
@@ -212,28 +212,51 @@ jobs:
       - run:
           command: ./.ci/validation.sh git-diff
 
+  run-inspections:
+    docker:
+      - image: nrmancuso/idea-docker:1.0.0
```

Comment:
Why not a container that is available already in open source?

Reply:
Couldn't find one that was up to date and/or didn't have bugs. This one is open source, too, with a reliable maintainer :)

Reply:
I am questioning reason of special docker creation. As it is already done by others.